geode-wan/src/main/java/org/apache/geode/cache/wan/internal/client/locator/LocatorHelper.java

```diff
@@ -12,11 +12,12 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
+
```

Comment:
Why did you add the space here (and in LocatorMembershipListenerImpl below)? I don't see other files with a space between the banner and package.

Reply:
I do it sometimes... habit... Like the separation of the unrelated comment block from the symbol. 🤷